src/declaration.c

```diff
@@ -375,7 +375,7 @@ void AliasDeclaration::semantic(Scope *sc)
     if (s && ((s->getType() && type->equals(s->getType())) || s->isEnumMember()))
         goto L2;                        // it's a symbolic alias
 
-    type = type->addStorageClass(storage_class);
+    type = type->addSTC(storage_class);
```

Comment:
Not possible. Built-in basic types (e.g. `int`) has already deco, so parsing const(int)` will run `Type::int32->addSTC(STCconst)`.